src/cuga/backend/cuga_graph/nodes/cuga_lite/executors/tests/test_code_executor.py

```
@@ -83,6 +86,31 @@ async def test_dangerous_import_blocked(mock_state):
     assert "not allowed" in str(exc_info.value)
 
 
+@pytest.mark.asyncio
+async def test_skills_mode_allows_non_allowlisted_import(mock_state, monkeypatch):
+    """Skills mode skips import allowlist so skill workflows can use extra packages."""
+    monkeypatch.setattr(settings.skills, "enabled", True)
+    code = "import json\ndata = json.dumps({'ok': True})\nprint(data)"
+    result, new_vars = await CodeExecutor.eval_with_tools_async(
```

Ruff (0.15.18)

[warning] Unpacked variable `new_vars` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)